scripts/opensearch-test-helpers/destination_connector/test-ingest-opensearch-output.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import sys
+
+from opensearchpy import OpenSearch
+
+N_ELEMENTS = 1404
+
+if __name__ == "__main__":
+    print("Connecting to the OpenSearch cluster.")
+    client = OpenSearch(
+        hosts=[{"host": "localhost", "port": 9200}],
+        http_auth=("admin", "admin"),
+        use_ssl=True,
+        verify_certs=False,
+        ssl_show_warn=False,
+    )
+    print(client.info())
+
+    count = int(client.count(index="ingest-test-destination")["count"])
+    try:
+        assert count == N_ELEMENTS
+    except AssertionError:
+        sys.exit(
+            "OpenSearch dest check failed:"
```

Comment:
does opensearch support vector embeddings?

Reply:
@rbiseck3 It looks like it does with a **k-NN index** plugin. (the plugin appears to be from Opensource themselves and might be already installed) Are you thinking that we do the test in a similar fashion to the pgvector test?

Reply:
According to this we need to somehow train a model to do the actual knn search. So it's probably not worth it for the test. Unlike pgvector, it is not ready to do a vector search right away. (as far as I can tell)

https://opster.com/guides/opensearch/opensearch-machine-learning/how-to-set-up-vector-search-in-opensearch/

Reply:
Well maybe it will work without a model...
https://opster.com/guides/opensearch/opensearch-machine-learning/opensearch-knn/

Reply:
Adding vector search test for Opensearch and Elasticsearch

Reply:
Removed Elasticsearch vector test. Ryan did it in another branch. Party time.

Reply:
awesome, thank you!